### ISSUE TITLE
Remove all RAILS_ENV={{ rails_env }}

### ISSUE THREAD

**Issue by elf-pavlik** [openfoodfoundation/ofn-install#145]
After #89 we don't need them if `unicorn_user` executes command

**Comment by elf-pavlik** [comment]
I'll take a look at this cleanup issue after we resolve all the issues which prevent playbooks to fully succeed.

I understood that #85 aimed at removing all that exporting and #89 didn't remove any of them. Maybe I misunderstood it and it only aimed at convenience of running commands manually rather than cleaning up tasks in playbooks.

**Comment by sauloperez** [comment]
Yes, cleaning up it's something that we need to do but we had more urgent issues at that moment, and we still do.

**Comment by mkllnk** [comment]
Nice to have but closing due to no activity.
